app/matplotlib.ipynb

```diff
@@ -164,7 +181,11 @@
    "source": [
     "data = np.random.normal(0, 1, 1000)\n",
     "\n",
-    "# write your code here\n"
+    "plt.hist(data, bins=50)\n",
+    "\n",
+    "plt.title(\"Data\")\n",
+    "plt.xlabel(\"X axis\")\n",
+    "plt.ylabel(\"Y axis\")\n"
```

Comment:
You should add `plt.show()` at the end of this cell to ensure the histogram is displayed in the notebook.